# Seminar: Gene Expression and Drug Repositioning Analysis

Scripts and tutorials for analyzing expression and drug repositioning data. Seminar for the module: <br>[QB1: Epidemiologie, Med. Biometrie u. Med. Informatik](https://lsf.uni-rostock.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=176669&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung&idcol=k_semester.semid&idval=20252&getglobal=semester&htmlBodyOnly=true)

---

## Description

These sessions will cover the use of a variety of software tools needed for the analysis of microbiome data, from the handling of the Illumina sequencing data, to the processing of 16S rRNA amplicon data. During these sessions, the students will be able to:

* Learn the basics on the linux / Mac OSX command line environment; 
* Study the common data formats in the analysis of gene expression
* Obtain gene expression data from Illumina sequencings;
* Identifying differentially expressed genes from expression experiments;
* Discovering the relevant pathways and biological processes from differentially expressed genes; and
* Finding drugs that reverse the molecular phenotypes in gene expression data. 

##### Venues and Dates

* Winter semester: November 2025


##### Presentations

- Download here: <!-- [link](https://github.com/barrantesisrael/mbtmicrobiome2023/blob/main/MBTPraktikum2024V01.pdf) <p> -->


##### Software and data

- All required data and scripts are available through our [gitlab repository](https://gitlab.uni-rostock.de/wb283/qb1rnaseq); all analyses are carried out in virtual [Google Colab](https://colab.research.google.com/) environments (google email account needed to login). 

- Open the experimental protocol here (Jupyter Notebook): [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/barrantesisrael/qb1rnaseq/blob/main/QB1_RNAseq_WS2025_20250714_V59.ipynb)

---

## Bibliography

1. **Anaconda Software Distribution (2020)**. Anaconda Documentation. Anaconda Inc. Available at: https://docs.anaconda.com/
2. **Anders, S. et al. (2010)**. Differential expression analysis for sequence count data. *Genome Biology*, 11(10), R106. https://doi.org/10.1186/gb-2010-11-10-r106
3. **Benjamini, Y. et al. (1995)**. Controlling the false discovery rate: a practical and powerful approach to multiple testing. *Journal of the Royal Statistical Society: Series B (Methodological)*, 57(1), 289-300. https://doi.org/10.1111/j.2517-6161.1995.tb02031.x
4. **Bisong, E. (2019)**. Google Colaboratory. In: Building Machine Learning and Deep Learning Models on Google Cloud Platform. Apress, Berkeley, CA. https://doi.org/10.1007/978-1-4842-4470-8_7
5. **Buffalo, V. (2015)**. Bioinformatics Data Skills: Reproducible and Robust Research with Open Source Tools. O'Reilly Media. ISBN: 978-1-449-36737-4
6. **Chen, Y. et al. (2016)**. From reads to genes to pathways: differential expression analysis of RNA-Seq experiments using Rsubread and the edgeR quasi-likelihood pipeline. *F1000Research*, 5, 1438. https://doi.org/10.12688/f1000research.8987.2
7. **Cock, P.J. et al. (2010)**. The Sanger FASTQ file format for sequences with quality scores, and the Solexa/Illumina FASTQ variants. *Nucleic Acids Research*, 38(6), 1767-1771. https://doi.org/10.1093/nar/gkp1137
8. **Conesa, A. et al. (2016)**. A survey of best practices for RNA-seq data analysis. *Genome Biology*, 17(1), 13. https://doi.org/10.1186/s13059-016-0881-8
9. **Consortium, S.M. (2014)**. A comprehensive assessment of RNA-seq accuracy, reproducibility and information content by the Sequencing Quality Control Consortium. *Nature Biotechnology*, 32(9), 903-914. https://doi.org/10.1038/nbt.2957
10. **Duan, Q. et al. (2016)**. L1000CDSÂ²: LINCS L1000 characteristic direction signatures search engine. *npj Systems Biology and Applications*, 2(1), 16015. https://doi.org/10.1038/npjsba.2016.15
11. **Ensemble Genome Browser Team (2021)**. The GTF (General Transfer Format) specification. Available at: https://www.ensembl.org/info/website/upload/gff.html
12. **Griffith, M. et al. (2015)**. Informatics for RNA-seq: a web resource for analysis on the cloud. *PLoS Computational Biology*, 11(8), e1004393. https://doi.org/10.1371/journal.pcbi.1004393
13. **Huber, W. et al. (2015)**. Orchestrating high-throughput genomic analysis with Bioconductor. *Nature Methods*, 12(2), 115-121. https://doi.org/10.1038/nmeth.3252
14. **Kluyver, T. et al. (2016)**. Jupyter Notebooks-a publishing format for reproducible computational workflows. *ELPUB*, 87-90. https://doi.org/10.3233/978-1-61499-649-1-87
15. **Leek, J.T. et al. (2010)**. Tackling the widespread and critical impact of batch effects in high-throughput data. *Nature Reviews Genetics*, 11(10), 733-739. https://doi.org/10.1038/nrg2825
16. **Love, M.I. et al. (2014)**. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. *Genome Biology*, 15(12), 550. https://doi.org/10.1186/s13059-014-0550-8
17. **Patro, R. et al. (2017)**. Salmon provides fast and bias-aware quantification of transcript expression. *Nature Methods*, 14(4), 417-419. https://doi.org/10.1038/nmeth.4197
18. **Pearson, W.R. et al. (1988)**. Improved tools for biological sequence comparison. *Proceedings of the National Academy of Sciences*, 85(8), 2444-2448. https://doi.org/10.1073/pnas.85.8.2444
19. **Pertea, M. et al. (2020)**. GFF utilities: GffRead and GffCompare. *F1000Research*, 9, 304. https://doi.org/10.12688/f1000research.23297.2
20. **Pushpakom, S. et al. (2018)**. Drug repurposing: progress, challenges and recommendations. *Nature Reviews Drug Discovery*, 18(1), 41-58. https://doi.org/10.1038/nrd.2018.168
21. **R Core Team (2024)**. R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. Available at: https://www.R-project.org/
22. **Raudvere, U. et al. (2019)**. g:Profiler: a web server for functional enrichment analysis and conversions of gene lists (2019 update). *Nucleic Acids Research*, 47(W1), W191-W198. https://doi.org/10.1093/nar/gkz369
23. **Subramanian, A. et al. (2005)**. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. *Proceedings of the National Academy of Sciences*, 102(43), 15545-15550. https://doi.org/10.1073/pnas.0506580102
24. **Subramanian, A. et al. (2017)**. A next generation connectivity map: L1000 platform and the first 1,000,000 profiles. *Cell*, 171(6), 1437-1452. https://doi.org/10.1016/j.cell.2017.10.049
25. **Tibshirani, R. (1988)**. Estimating transformations for regression via additivity and variance stabilization. *Journal of the American Statistical Association*, 83(402), 394-405. https://doi.org/10.1080/01621459.1988.10478610
26. **Wickham, H. (2016)**. ggplot2: Elegant Graphics for Data Analysis. Springer-Verlag New York. ISBN: 978-3-319-24277-4
27. **Wickham, H. et al. (2023)**. dplyr: A Grammar of Data Manipulation. R package version 1.1.4. Available at: https://CRAN.R-project.org/package=dplyr
28. **Wickham, H. et al. (2023)**. tidyr: Tidy Messy Data. R package version 1.3.0. Available at: https://CRAN.R-project.org/package=tidyr
29. **Williams, A.G. et al. (2014)**. RNA-seq data: challenges in and recommendations for experimental design and analysis. *Current Protocols in Human Genetics*, 83(1), 11-13. https://doi.org/10.1002/0471142905.hg1113s83
  
---

## Contact

Dr. rer. nat. Israel Barrantes <br>
Research Group Translational Bioinformatics (head)<br>
Institute for Biostatistics and Informatics in Medicine and Ageing Research, Office 3017<br>
Rostock University Medical Center<br>
Ernst-Heydemann-Str. 8<br>
18057 Rostock, Germany<br>

Email: israel.barrantes[bei]uni-rostock.de

---
Last update 2025/07/23
