# Seminar: Gene Expression and Drug Repositioning Analysis

Scripts and tutorials for analyzing expression and drug repositioning data. Seminar for the module: <br>[QB1: Epidemiologie, Med. Biometrie u. Med. Informatik](https://lsf.uni-rostock.de/qisserver/rds?state=verpublish&status=init&vmfile=no&publishid=176669&moduleCall=webInfo&publishConfFile=webInfo&publishSubDir=veranstaltung&idcol=k_semester.semid&idval=20252&getglobal=semester&htmlBodyOnly=true)

---

## Description

These sessions will cover the use of a variety of software tools needed for the analysis of microbiome data, from the handling of the Illumina sequencing data, to the processing of 16S rRNA amplicon data. During these sessions, the students will be able to:

* Learn the basics on the linux / Mac OSX command line environment; 
* Study the common data formats in the analysis of gene expression
* Obtain gene expression data from Illumina sequencings;
* Identifying differentially expressed genes from expression experiments;
* Discovering the relevant pathways and biological processes from differentially expressed genes; and
* Finding drugs that reverse the molecular phenotypes in gene expression data. 

##### Venues and Dates

* Winter semester: November 2025


##### Presentations

- Download here: [link](https://github.com/barrantesisrael/mbtmicrobiome2023/blob/main/MBTPraktikum2024V01.pdf) <p>


##### Software and data

- All required data and scripts are available through our [gitlab repository](https://gitlab.uni-rostock.de/wb283/qb1rnaseq); all analyses are carried out in virtual [Google Colab](https://colab.research.google.com/) environments (google email account needed to login). 

- Open the experimental protocol here (Jupyter Notebook): [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/barrantesisrael/qb1rnaseq/blob/main/QB1_RNAseq_WS2025_20250714_V58.ipynb)

---

## Bibliography

* Callahan BJ et al. (2016). Bioconductor workflow for microbiome data analysis: from raw reads to community analyses. _F1000Research_, 5, 1492. https://doi.org/10.12688/f1000research.8986.1
* Comeau AM et al. (2017). Microbiome Helper: a Custom and Streamlined Workflow for Microbiome Research. _mSystems_, 2(1): e00127-16; https://doi.org/10.1128/mSystems.00127-16
* DeSantis TZ et al. (2006) Greengenes, a chimera-checked 16S rRNA gene database and workbench compatible with ARB. _Appl Environ Microbiol_ Jul;72(7):5069-72. https://journals.asm.org/doi/10.1128/aem.03006-05
* Gentleman RC et al. (2004). Bioconductor: open software development for computational biology and bioinformatics. _Genome Biology_, 5(10), R80. https://doi.org/10.1186/gb-2004-5-10-r80
* Goodrich JKK et al. (2014). Conducting a Microbiome Study. _Cell_, 158(2), 250–262. https://doi.org/10.1016/j.cell.2014.06.037
* Knight R et al. (2018). Best practices for analysing microbiomes. _Nat Rev Microbiol_, 16, 410–422. https://doi.org/10.1038/s41579-018-0029-9
* Kozich JJ, Westcott SL, Baxter NT, Highlander SK, Schloss PD. (2013) Development of a dual-index sequencing strategy and curation pipeline for analyzing amplicon sequence data on the MiSeq Illumina sequencing platform. _Appl Environ Microbiol_, 79(17):5112-20. http://doi.org/10.1128/AEM.01043-13
* Lu, J., Rincon, N., Wood, D.E. et al. (2022) Metagenome analysis using the Kraken software suite. _Nat Protoc._ https://doi.org/10.1038/s41596-022-00738-y
* Masella AP et al. (2012). PANDAseq: Paired-end assembler for illumina sequences. _BMC Bioinformatics_, 13(1), 31. https://doi.org/10.1186/1471-2105-13-31
* McDonald D et al. (2012). The Biological Observation Matrix (BIOM) format or: how I learned to stop worrying and love the ome-ome. _GigaScience_, 1(1), 7. https://doi.org/10.1186/2047-217X-1-7
* McMurdie PJ and Holmes S. (2013). Phyloseq: An R Package for Reproducible Interactive Analysis and Graphics of Microbiome Census Data. _PLoS ONE_, 8(4), e61217. https://doi.org/10.1371/journal.pone.0061217
* Poolman TM, Townsend-Nicholson A, Cain A. (2022) Teaching genomics to life science undergraduates using cloud computing platforms with open datasets. _Biochem Mol Biol Educ._ 50(5):446-449. https://doi.org/10.1002/bmb.21646
* Quast C et al. (2013) The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. _Nucl. Acids Res._ 41 (D1): D590-D596. https://doi.org/10.1093/nar/gks1219

  
---

## Contact

Dr. rer. nat. Israel Barrantes <br>
Junior Research Group Translational Bioinformatics (head)<br>
Institute for Biostatistics and Informatics in Medicine and Ageing Research, Office 3017<br>
Rostock University Medical Center<br>
Ernst-Heydemann-Str. 8<br>
18057 Rostock, Germany<br>

Email: israel.barrantes[bei]uni-rostock.de

---
Last update 2025/07/18
